frontends/web/src/containers/SubmitModel.js

```diff
@@ -0,0 +1,194 @@
+ */
+
+import React, { useState, useEffect, useContext } from "react";
+import { Button, Col, Container, Row } from "react-bootstrap";
+import UserContext from "../containers/UserContext";
+
+const SubmitModel = (props) => {
+  const context = useContext(UserContext);
+
+  const [inputFile, setInputFile] = useState(null);
+  const [loadingFile, setLoadingFile] = useState(true);
+  const [loading, setLoading] = useState(true);
+
+  useEffect(() => {
+    setInputFile(document.getElementById("input-file"));
+  }, []);
+
+  const handleUploadModel = (e) => {
+    inputFile?.click();
+    const fetchData = async () => {
+      await context.api.getTask(props.match.params.taskCode);
+    };
+    fetchData();
+    const task = fetchData();
+    setLoadingFile(false);
+    console.log(task);
+  };
+
+  const handleSubmitModel = (e) => {
+    e.preventDefault();
+
+    if (inputFile.files.length !== 0) {
+      setLoading(false);
+      const user = context.api.getCredentials();
+      const file = inputFile.files[0];
+      const fetchData = async () => {
+        await context.api.uploadModelUser(
+          file,
+          user.username,
+          file.name,
+          file.type,
+          user.id,
+          props.match.params.taskCode
+        );
+      };
+      const result = fetchData();
+      setLoading(true);
+    } else {
+      alert("Please upload a model");
+      setLoadingFile(true);
+    }
+  };
+  return (
+    <>
+      {loading ? (
+        <Container className="mb-5 pb-5">
+          <Col className="m-auto" lg={9}>
+            <div className="mb-5 text-center">
+              <h1 className="my-4 pt-3 text-uppercase">
+                Upload your own model
+              </h1>
+              <h2 className="task-page-header d-block font-weight-normal m-0 text-reset">
+                Imagine you developed a fancy model, living in
+                /path/to/fancy_project, and you want to share your amazing
+                results on the Dynabench model leaderboard. Dynalab makes it
+                easy for you to do just that
+              </h2>
+            </div>
+            <div className="mt-5">
+              <ul>
+                <li>
+                  <strong>Download dynalab 2.0</strong>
+                  <p>
+                    Please download{" "}
+                    <a
+                      download
+                      href="https://models-dynalab.s3.eu-west-3.amazonaws.com/MT/dynalab-base-mt.zip"
+                    >
+                      {" "}
+                      this
+                    </a>{" "}
+                    project, and unzip in your own computer
+                  </p>
+                </li>
+                <li>
+                  <strong>Understand the folders</strong>
+                  <p>
+                    Inside the repository you will find a folder called app.
+                    Inside this folder you will find a folder called domain.
```

Comment:
It's unclear from these instructions what "app" and "app/domain" are for. It's also unclear if model.py lives in app/domain/ or somewhere else